ci/build.sh

```diff
@@ -9,10 +9,19 @@ dn=$(dirname $0)
 pkg_install sudo which attr fuse bison dbus-daemon \
     libubsan libasan libtsan clang python2 \
     elfutils git gettext-devel libappstream-glib-devel hicolor-icon-theme \
-    dconf-devel fuse-devel \
+    dconf-devel fuse-devel meson dbus-devel \
```

Comment:
We're not building CI this way anymore. Now it happens in .github/workflows/check.yml, but lets take that in a followup.

Reply:
Attempted to be fixed in #3150.